core/commonMain/src/Instant.kt

```diff
@@ -303,6 +333,18 @@ public fun Instant.plus(unit: DateTimeUnit.TimeBased): Instant =
  */
 public expect fun Instant.plus(value: Int, unit: DateTimeUnit, timeZone: TimeZone): Instant
 
+/**
+ * Returns an instant that is the result of subtracting the [value] number of the specified [unit] from this instant
+ * in the specified [timeZone].
+ *
+ * If the [value] is positive, the returned instant is earlier than this instant.
+ * If the [value] is negative, the returned instant is later than this instant.
+ *
+ * @throws DateTimeArithmeticException if this value or the result is too large to fit in [LocalDateTime].
+ */
+public fun Instant.minus(value: Int, unit: DateTimeUnit, timeZone: TimeZone): Instant =
+    plus(-value, unit, timeZone)
```

Comment:
Is the negation always overflow-safe? I.e. it certainly can overflow at MIN_VALUE, but would this overflow be compensated later?

Reply:
You're right, this would be silently wrong. I pushed a fix for this and a similar problem with `LocalDate.minus(Int, ...)` on JVM.

Reply:
Could you also ensure that `instant.minus(Long.MIN_VALUE, DateTimeUnit.NANOSECOND) > instant`?

Reply:
Done. Since this edge case is a requirement, consistency also required solving a whole new can of worms with `DateTimePeriod`, which I also did.